integrations/ollama/pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai", "ollama"]
+dependencies = ["haystack-ai", "ollama>=0.4.0"]
```

Comment:
Should we also remove python 3.8 check:
`"Programming Language :: Python :: 3.8",`

Reply:
makes sense. Done!